recipes/gatk4/meta.yaml

```diff
@@ -12,67 +12,99 @@ source:
 
 build:
   noarch: generic
-  number: 0
+  number: 1
   run_exports:
      - {{ pin_subpackage('gatk4', max_pin="x") }}
 
-requirements:
-  host:
-    - r-base =3.6
-    - r-gplots
-    - r-ggplot2
-    - r-gsalib
-    - r-reshape
-    - gcnvkernel =0.8
-  run:
-    - openjdk >=17,<18
-    - python
-    - r-base =3.6
-    - r-gplots
-    - r-ggplot2
-    - r-gsalib
-    - r-reshape
-    - gcnvkernel =0.8
-
 test:
   commands:
+    - java -version
     - gatk -h
     - gatk --list
     - gatk HaplotypeCaller --help
-    - gatk DetermineGermlineContigPloidy --help
+  imports:
+    - gcnvkernel
 
+#All packages are listed below following GATK's template, except for tensorflow. 
+#The specified version 1.15.0 of tensorflow results in an issue as described in this link: https://github.com/bioconda/bioconda-recipes/issues/45600.outputs:
 outputs:
   - name: gatk4
     script: build_main.sh
     requirements:
+      build:
+        - conda-forge::gcc_impl_linux-64=7.5.0=habd7529_20
```

Comment:
you don't need to prefix with conda-forge ... conda-forge is used by default with strict channel priority. This means we will take at first packages from conda-forge, then bioconda and ignore the all others.